src/main/java/HypeBot.java

```diff
@@ -0,0 +1,230 @@
+     *
+     * @param task Task to add to TASKS.
+     */
+    private static void add(Task task) {
+        TASKS.add(task);
+        System.out.println(addBufferLine("HECK YEAH, ADDED: "
+                + task
+                + "!\nYOU'VE NOW GOT "
+                + TASKS.size()
+                + " TASKS TO GO!\n"));
+    }
+
+    /**
+     * Takes in task number and marks corresponding task as incomplete.
+     * If task number not found in list, throws IndexOutOfBoundsException.
+     *
+     * @param idx Task number to mark incomplete.
+     * @throws IndexOutOfBoundsException Thrown if task number invalid (too low / too high).
+     */
+    private static void unmark(int idx) throws IndexOutOfBoundsException {
+        TASKS.get(idx).unmark();
+        System.out.println(addBufferLine("AIGHT, LET'S GET READY TO CONQUER THIS TASK:\n  "
+                + TASKS.get(idx)
+                + "\n"));
+    }
+
+    /**
+     * Takes in task number and marks corresponding task as complete.
+     * If task number not found in list, throws IndexOutOfBoundsException.
+     *
+     * @param idx Task number to mark complete.
+     * @throws IndexOutOfBoundsException Thrown if task number invalid (too low / too high).
+     */
+    private static void mark(int idx) throws IndexOutOfBoundsException {
+        TASKS.get(idx).mark();
+        System.out.println(addBufferLine("AIGHT, ABSOLUTELY CONQUERED THIS TASK:\n  "
+                + TASKS.get(idx)
+                + "\n"));
+    }
+
+    /**
+     * Takes in task number and deletes corresponding task from TASKS.
+     * If task number not found in list, throws IndexOutOfBoundsException.
+     *
+     * @param idx Task number to delete.
+     * @throws IndexOutOfBoundsException Thrown if task number invalid (too low / too high).
+     */
+    private static void delete(int idx) throws IndexOutOfBoundsException {
+        Task removed = TASKS.remove(idx);
+        System.out.println(addBufferLine("Say no more, BABY BYE BYE BYE to this task:\n "
+            + removed
+            + "!\nYOU'VE NOW GOT "
+            + TASKS.size()
+            + " TASKS TO GO!\n"));
+    }
+
+    public static void main(String[] args) {
+        greet();
+
+        Scanner sc = new Scanner(System.in);
+        boolean canExit = false;
+
+        while (!canExit) {
+            String line = sc.nextLine();
+
+            if (line.trim().isEmpty()) {
+                continue;
+            }
+
+            String[] splitLineForDates = line.split("/");
+            String[] commandAndTaskName = splitLineForDates[0].split(" ");
+            String command = commandAndTaskName[0];
+            StringBuilder taskNameBuilder = new StringBuilder();
+            for (int i = 1; i < commandAndTaskName.length; i++) {
+                taskNameBuilder.append(commandAndTaskName[i]).append(" ");
+            }
+            String taskName = taskNameBuilder.toString();
+
+            switch (command) {
+                case "bye":
```

Comment:
Oops, case should not have indentation according to CS2103T coding standard. Similar goes across the switch statements.